docs/_releases/v2.0.0-pre.4/matchers.md

```diff
@@ -112,6 +112,26 @@ Requires the value to be a `Function`.
 Requires the value to be an `Array`.
 
 
+#### `sinon.match.array.deepEquals(iterable)`
+
+Requires an iterable value to be deep equal another one.
```

Comment:
Of course!
I'm already working on the `Map` and `Set` matchers, for now I'll make sure to add commits allowing arrays only on the matchers we already have and update the docs.